openmetadata-ui/src/main/resources/ui/src/constants/constants.ts

```diff
@@ -54,6 +54,7 @@ export const USER_DATA_SIZE = 5;
 export const INITIAL_PAGING_VALUE = 1;
 export const JSON_TAB_SIZE = 2;
 export const KNOWLEDGE_LIST_LENGTH = 8;
+export const PAGE_SIZE_SMALL = 5;
```

Comment:
Why we want to support PAGE_SIZE_SMALL

Reply:
Wanted to limit the results to 5 for the widgets listing. I'll use an existing constant for that and remove this one.